### ISSUE TITLE
Saved data corrupted? From playing game as administrator on desktop shocut

### ISSUE THREAD

**Issue by AwkAlt** [hovgaardgames/startupcompany#570]
I've run my gams as administrator since the developer recommended it due to my games lagging issues, it has fix that. Since last night I've been playing the game from the created shortcut on my desktop and being run as administrator. Since last night, I've gone from day 121 to this afternoon day 250 with $10M, I've closed the game since I figured I should get some rest, and when I tried to acess the game normally through steam, it opened the save from Day 121 and not the save data that I had from Day 250. Now I try to go back to the created shortcut on my desktop and run it as an administrator. I couldn't access the saved data from day 250 anymore. I was wondering if theres anyway to recover that :( I had a lot of progress in that game, it would suck if it never got saved or something.

TL;DR: Played the game on shortcut desktop running as administrator. Tried playing back on steam, saw that the saved data of the game through shortcut desktop accessed is gone and might have gotten override from the saved data from steam. Asking for help if there's anyway I can get my saved data progress back from the shortcut on desktop.

**Comment by MazeBank** [comment]
Hi @AwkAlt 
Thank you for reporting it on GITHUB as asked.
This issue will be noted under bugs soon.

If you have any further questions? Feel free to ask!

<Jason/t>

**Comment by jhovgaard** [comment]
This issue should have been fixed with the atomic saving feature.

Thanks for reporting!
